packs/journals/ancestries.json

```diff
@@ -855,7 +855,7 @@
orship Tsukiyo, knowing he finds loud worship agitating, and they take care to not invoke his name or imagery during harvest moon or lunar new year celebrations to ensure their cheers and belly drums don't draw his attention. Tanuki with Inner Sea cultural traditions tend to revere Cayden Cailean, seeing the blood of a tanuki in his commoner status (and drunken accomplishments), with some tanuki statues even depicting Cayden with a big belly and leaf atop his head to suggest that he was simply a tanuki in human form, tricking the Avistani even to this day.</p><p>Tanuki generally use their powers to humble the rich and snobby; at the very least, they create illusions with no more evil in their hearts than a simple joke (or a free meal taken from one who can afford it), rather than acting out of malice. However, there are some tanuki do harbor resentment, which can take root and fester in their hearts. These tanuki turn to cruel ends, known for all manner of nefarious deeds, chief among them cooking their enemies into soups or stews once they've finished tricking them.</p><p><strong>Popular Edicts</strong></p><p>Choose excitement over stability, let bygones be bygones, make your community laugh</p><p><strong>Popular Anathema</strong></p><p>Feel entitled to social status, prank one of lower means than yourself, wallow in your failures</p><h2>Tanuki Mechanics</h2><p><strong>Hit Points</strong> 10</p><p><strong>Speed</strong> 25 feet</p><p><strong>Attribute Boosts</strong> Constitution, Charisma, Free</p><p><strong>Attribute Flaw</strong> Wisdom</p><p><strong>Languages</strong> Common, Tanuki</p><p><strong>Additional Languages</strong> equal to your Intelligence modifier (if it's positive). Choose from Elven, Fey, Gnomish, Goblin, Kitsune, Tengu, and any other languages to which you have access (such as the languages prevalent in your region)</p><p><strong>Change Shape</strong> <span class=\"pf2-icon\">1</span> (concentrate, polymorph, primal, tanuki) You can transform into a mundane raccoon dog, using the statistics of pest form. This is a specific raccoon dog form that's the same age and body type as your true form and has roughly analogous physical traits, such as hair color. Using Change Shape counts as creating a disguise for the Impersonate use of Deception. You lose any unarmed Strikes you gained from a tanuki heritage or ancestry feat in this form. You can remain in your raccoon dog form indefinitely, and you can shift back to your tanuki form by using this action again.</p><h2>Tanuki Heritages</h2><p>A tanuki's heritage reflects a unique virtue that beats in the tanuki's heart. Choose one of the following tanuki heritages at 1st level.</p><p>@UUID[Compendium.pf2e.heritages.Item.Ascetic Tanuki]</p><p>@UUID[Compendium.pf2e.heritages.Item.Courageous Tanuki]</p><p>@UUID[Compendium.pf2e.heritages.Item.Even-Tempered Tanuki]</p><p>@UUID[Compendium.pf2e.heritages.Item.Steadfast Tanuki]</p><p>@UUID[Compendium.pf2e.heritages.Item.Virtuous Tanuki]</p>",
```

Comment:
Consider adding`<strong>Size</strong> Small` to the Mechanics.

Reply:
Good catch - I also added size to the Hobgoblin entry as that's the one I looked at as a basis and was also missing the size data point.